java/Cherepnin_Oleh/src/com/zoolatech/lecture6/tasks/_1/City.java

```diff
@@ -0,0 +1,49 @@
+package com.zoolatech.lecture6.tasks._1;
+
+import java.util.Objects;
+
+public class City {
+    private String name;
+    private String country;
+    private int population;
+
+    public City(String name, String country, int population) {
+        this.name = name;
+        this.country = country;
+        this.population = population;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
methods `equals`, `hashCode` and `toString` are usually defined at the end of a class